### PR TITLE
Intermediate Global Taiko Showdown 2018 Wiki Page

### PR DESCRIPTION
**Pull request by fajar13k**
### Description
Start IGTS2018 wiki page. This page will be frequently updated before and during the duration of the tournament take place.

### Status
Pending Review

### Note
Need help reviewing it @Joehuu @WalterToro

### CODE REVIEW

File: wiki/Tournaments/IGTS/2018/en.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+tags:
+- IGTS 2018
+- IGTS2018
+---
+Intermediate Global Taiko Showdown 2018
+=============================
+
+![Intermediate Global Taiko Showdown 2018](banner.jpg)
+
+**Intermediate Global Taiko Showdown 2018** is a country-based double elimination two-on-two osu!taiko tournament hosted by ![][flag_CA] [FlamingRok](/users/263110) and ![][flag_FR] [Kasumii-sama](/users/6177263). It is the first iteration of the tournament.
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: Add `The` in the beginning.

File: wiki/Tournaments/IGTS/2018/en.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+## Mappool
+
+*The mappools will be revealed one week in advance for each stage.*
+
+------------------------------------------------------------------------
+
+## Ruleset
+### General Rules
+
+1. Intermediate Global Taiko Showdown 2018 is a two-on-two Team Vs tournament, with the aspect of double-elimination bracket.
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: Add `The` in the beginning.

File: wiki/Tournaments/IGTS/2018/en.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+2. In order for your registration to count, you must meet following conditions:
+   - Participant must fill out the provided form.
+   - Participant must be within rank **3,500 - 10,000 during registrations.**
+   - Participant are not allowed to rank up further than rank **2,500 during the tournament's occurrence**
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: Add period in the end.

File: wiki/Tournaments/IGTS/2018/en.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 1 months of osu!supporter tag for each team member |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 1 months of osu!supporter tag for each team member |
+
+## Organization
+
+The Intermediate Global Taiko Showdown 2018 is run by various osu!taiko community members.
+
+| Position | Members |
+| --- | --- |
+| Organizers | ![][flag_CA] [FlamingRok](/users/2015353), ![][flag_FR] [Kasumii-sama](/users/6177263) |
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: `Organizers` -> `Organisers`

File: wiki/Tournaments/IGTS/2018/en.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+
+## Prizes
+
+| Placing | Prize(s) |
+| --- | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 3 months of osu!supporter tag and badge for each team member |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 1 months of osu!supporter tag for each team member |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 1 months of osu!supporter tag for each team member |
+
+## Organization
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: `Organization` -> `Organisation`

File: wiki/Tournaments/IGTS/2018/en.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+-----------------------
+
+## Mappool
+
+*The mappools will be revealed one week in advance for each stage.*
+
+------------------------------------------------------------------------
+
+## Ruleset
+### General Rules
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: Add a line between the two headers.

File: wiki/Tournaments/IGTS/2018/en.md [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+
+1. The Intermediate Global Taiko Showdown 2018 is a two-on-two Team Vs tournament, with the aspect of double-elimination bracket.
+
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: Remove this line.

File: wiki/Tournaments/IGTS/2018/en.md [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+2. In order for your registration to count, you must meet following conditions:
+   - Participant must fill out the provided form.
+   - Participant must be within rank **3,500 - 10,000 during registrations.**
+   - Participant are not allowed to rank up further than rank **2,500 during the tournament's occurrence.**
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: `are` -> `is`

File: wiki/Tournaments/IGTS/2018/en.md [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+   - Participant must join tournament's Discord server.
+3. The mapset will be announced the week before a stage begins.
+4. Match schedule is decided by the organizers, which will announce schedule the same day as the mapset every week. Each stage of this tournament will last a week, and that week starts when the mapset is announced a day before, so players need to get their match in before the week is over.
+5. If both team gets a tie on the TieBreaker, the TieBreaker will be replayed with FreeMod applied.
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: `team gets` -> `teams get`

File: wiki/Tournaments/IGTS/2018/en.md [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+6. If a player disconnects from the game, the opponent wins the round. 
+   - If the player disconnects during the first five seconds of the song, the match will be restarted.
+7. Maps cannot be used twice in the same round.
+8. If a player doesn't show up within **15 minutes** of the start time, the enemy wins by default.
+9.  Any other rules changes or unexpected occurrences will be announced in the tournament's forum thread.
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: Remove the extra space.

File: wiki/Tournaments/IGTS/2018/en.md [hunk 11]
```diff
@@ -0,0 +1,141 @@
...
+
+### Stage Regulations
+
+1. There are four stages to this tournament: Group Stage, Quarterfinals, Semifinals, and Finals.
+
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: Remove this line.

File: wiki/Tournaments/IGTS/2018/en.md [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+2. Matches are seeded. Teams will be seeded by performance points.
+3. In Group Stage, you need to win 4 beatmaps to win a match. (Best-of-7)
+4. In Round of 16, Quarterfinals, and Semifinals, you need to win 5 beatmaps to win a match. (Best-of-9)
+5. In Finals, you need to win 6 beatmaps to win a match. (Best-of-11)
+
+### Match Regulations
+1. A private lobby will be created for the match, and both teams will be invited into their match following the agreed schedule. Match will be played as **TeamVs and ScoreV2.**
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: Add a line above this.

File: wiki/Tournaments/IGTS/2018/en.md [hunk 13]
```diff
@@ -0,0 +1,141 @@
...
+
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: Remove this line.

File: wiki/Tournaments/IGTS/2018/en.md [hunk 14]
```diff
@@ -0,0 +1,141 @@
...
+2. Each team is allowed to select one warm-up map.
+   - During Group Stage warm-ups must not exceed 4:00 minutes. In the following stages the warm-up can be up to 6 minutes long if both teams agree.
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: `Group Stage warm-ups` -> `Group Stage, warm-ups`
`following stages the` -> `following stages, the`

File: wiki/Tournaments/IGTS/2018/en.md [hunk 15]
```diff
@@ -0,0 +1,141 @@
...
+   - Offensive songs may be blocked by referees.
+3. Map selection will be done by the team who is currently losing. At the start however, it will be whoever `!roll` the highest. When the starting team is decided, both teams will select one map to ban that is not the tiebreaker.
+4. In case of a tie, the tiebreaker will be played.
+5. Results will be displayed on Challonge, and announced by the referee.
+
+### Mappool Regulations
+
+1. Each stage will receive its own mappool, and it will progressively become harder.
+
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: Remove this line.

File: wiki/Tournaments/IGTS/2018/en.md [hunk 16]
```diff
@@ -0,0 +1,141 @@
...
+   - 2 FreeMod
+   - 1 Tiebreaker
+5. The tiebreaker will be played under NoMod conditions.
+   - In case of a tie, the tiebreaker will be replayed again with FreeMod activated.
+6. The mappool will be released after the last match of the previous stage has been played.
+
+### Scheduling Regulations
+
+1. Every stage is held **during the weekends.**
+
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: Remove this line.

File: wiki/Tournaments/IGTS/2018/en.md [hunk 17]
```diff
@@ -0,0 +1,141 @@
...
+4. In case of a tie, the tiebreaker will be played.
+5. Results will be displayed on Challonge, and announced by the referee.
+
+### Mappool Regulations
+
+1. Each stage will receive its own mappool, and it will progressively become harder.
+
+2. Each mappool consists of 6 brackets: NoMod, Hidden, HardRock, DoubleTime, FreeMod, and TieBreaker.
+3. Each mappool consists of 13 maps in total.
+4. The mappool consists of following format:
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: `of following` -> `of the following`

File: wiki/Tournaments/IGTS/2018/en.md [hunk 18]
```diff
@@ -0,0 +1,141 @@
...
+
+1. The Intermediate Global Taiko Showdown 2018 is a two-on-two Team Vs tournament, with the aspect of double-elimination bracket.
+
+2. In order for your registration to count, you must meet following conditions:
+   - Participant must fill out the provided form.
+   - Participant must be within rank **3,500 - 10,000 during registrations.**
+   - Participant are not allowed to rank up further than rank **2,500 during the tournament's occurrence.**
+   - Participant must join tournament's Discord server.
+3. The mapset will be announced the week before a stage begins.
+4. Match schedule is decided by the organizers, which will announce schedule the same day as the mapset every week. Each stage of this tournament will last a week, and that week starts when the mapset is announced a day before, so players need to get their match in before the week is over.
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: `organizers` -> `organisers`

File: wiki/Tournaments/IGTS/2018/en.md [hunk 19]
```diff
@@ -0,0 +1,141 @@
...
+4. In Round of 16, Quarterfinals, and Semifinals, you need to win 5 beatmaps to win a match. (Best-of-9)
+5. In Finals, you need to win 6 beatmaps to win a match. (Best-of-11)
+
+### Match Regulations
+1. A private lobby will be created for the match, and both teams will be invited into their match following the agreed schedule. Match will be played as **TeamVs and ScoreV2.**
+
+2. Each team is allowed to select one warm-up map.
+   - During Group Stage warm-ups must not exceed 4:00 minutes. In the following stages the warm-up can be up to 6 minutes long if both teams agree.
+   - Offensive songs may be blocked by referees.
+3. Map selection will be done by the team who is currently losing. At the start however, it will be whoever `!roll` the highest. When the starting team is decided, both teams will select one map to ban that is not the tiebreaker.
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: `tiebreaker` -> `TieBreaker`

Note the capitalisation on the other occurrences.

File: wiki/Tournaments/IGTS/2018/en.md [hunk 20]
```diff
@@ -0,0 +1,139 @@
...
+tags:
+- IGTS 2018
+- IGTS2018
+---
+Intermediate Global Taiko Showdown 2018
+=============================
+
+![Intermediate Global Taiko Showdown 2018](banner.jpg)
+
+The **Intermediate Global Taiko Showdown 2018** is a country-based double elimination two-on-two osu!taiko tournament hosted by ![][flag_CA] [FlamingRok](/users/263110) and ![][flag_FR] [Kasumii-sama](/users/6177263). It is the first iteration of the tournament.
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: `the tournament` -> `this tournament`

File: wiki/Tournaments/IGTS/2018/en.md [hunk 21]
```diff
@@ -0,0 +1,139 @@
...
+
+1. The Intermediate Global Taiko Showdown 2018 is a two-on-two Team Vs tournament, with the aspect of double-elimination bracket.
+2. In order for your registration to count, you must meet following conditions:
+   - Participant must fill out the provided form.
+   - Participant must be within rank **3,500 - 10,000 during registrations.**
+   - Participant is not allowed to rank up further than rank **2,500 during the tournament's occurrence.**
+   - Participant must join tournament's Discord server.
+3. The mapset will be announced the week before a stage begins.
+4. Match schedule is decided by the organisers, which will announce schedule the same day as the mapset every week. Each stage of this tournament will last a week, and that week starts when the mapset is announced a day before, so players need to get their match in before the week is over.
+5. If both teams gets a tie on the TieBreaker, the TieBreaker will be replayed with FreeMod applied.
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: `gets` -> `get`

File: wiki/Tournaments/IGTS/2018/en.md [hunk 22]
```diff
@@ -0,0 +1,139 @@
...
+6. If a player disconnects from the game, the opponent wins the round. 
+   - If the player disconnects during the first five seconds of the song, the match will be restarted.
+7. Maps cannot be used twice in the same round.
+8. If a player doesn't show up within **15 minutes** of the start time, the enemy wins by default.
+9. Any other rules changes or unexpected occurrences will be announced in the tournament's forum thread.
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: `rules changes or unexpected` -> `rules, changes, or unexpected`

File: wiki/Tournaments/IGTS/2018/en.md [hunk 23]
```diff
@@ -0,0 +1,139 @@
...
+1. Each stage will receive its own mappool, and it will progressively become harder.
+2. Each mappool consists of 6 brackets: NoMod, Hidden, HardRock, DoubleTime, FreeMod, and TieBreaker.
+3. Each mappool consists of 13 maps in total.
+4. The mappool consists of the following format:
+   - 4 NoMod
+   - 2 Hidden
+   - 2 HardRock
+   - 2 DoubleTime
+   - 2 FreeMod
+   - 1 Tiebreaker
```
Comment (Joehuu) [inline wiki/Tournaments/IGTS/2018/en.md]: `Tiebreaker` -> `TieBreaker`

File: wiki/Tournaments/IGTS/2018/en.md [hunk 24]
```diff
@@ -0,0 +1,139 @@
...
+5. If both teams get a tie on the TieBreaker, the TieBreaker will be replayed with FreeMod applied.
+6. If a player disconnects from the game, the opponent wins the round. 
+   - If the player disconnects during the first five seconds of the song, the match will be restarted.
+7. Maps cannot be used twice in the same round.
+8. If a player does not show up within **15 minutes** of the start time, the enemy wins by default.
+9. Any other rules, changes, or unexpected occurrences will be announced in the tournament's forum thread.
+
+### Stage Regulations
+
+1. There are four stages to this tournament: Group Stage, Quarterfinals, Semifinals, and Finals.
```
Comment (WalterToro) [inline wiki/Tournaments/IGTS/2018/en.md]: Looking below, isnt this like five stages? (Round of 16 is missing)

Comment (fajar13k) [reply wiki/Tournaments/IGTS/2018/en.md]: Right, the tournament host's original schedule was so confusing lol. I'll fix asap.

**Pull request merged**